UiPath.FreeRdpClient/UiPath.FreeRdpClient/UiPath.FreeRdpClient.csproj

```diff
@@ -12,6 +12,7 @@
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
```

Comment:
This is included with the next PackageReference: Microsoft.Extensions.Hosting.Abstractions